web/app/js/components/MetricsTable.jsx

```diff
@@ -72,7 +72,7 @@ const columnDefinitions = (sortable = true, resource, ConduitLink) => {
 
 const numericSort = (a, b) => (_.isNil(a) ? -1 : a) - (_.isNil(b) ? -1 : b);
 
-export default class TabbedMetricsTable extends React.Component {
+export default class MetricsTable extends React.Component {
```

Comment:
Meant to mention this yesterday -- one more thing that came up in review was that we wanted the default sort order for this table to be alphabetically by deployment name (first column). Can I bug you to make that change as part of this branch?